app/pages/profile/index.cjsx

```diff
@@ -35,6 +43,7 @@ UserProfilePage = React.createClass
     @getProfileHeader(@props.profileUser)
 
   componentWillReceiveProps: (nextProps) ->
+    document.documentElement.classList.add 'on-secondary-page'
```

Comment:
Shouldn't be, and yet, I'm sometimes seeing the recent comments / stats pages load up without styling, so i added this as a catch-all. not sure why that is happening. i'll take another look